apps/meteor/client/views/room/composer/messageBox/hooks/useMessageBoxAutoFocus.ts

```diff
@@ -29,6 +29,11 @@ export const useMessageBoxAutoFocus = (enabled: boolean): Ref<HTMLElement> => {
 				return;
 			}
 
+			// This prevents focus from being stolen between the two RichText Composers
+			if ((target as HTMLElement).tagName === 'DIV' && (target as HTMLElement).isContentEditable) {
```

Comment:
Instead of checking for the element type, we could check if the experimental composer is active, using the feature preview flag, this way it is clearer to anyone reading the code why we do this